lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
+var set = require('lodash.set');
+var isString = require('lodash.isstring');
+var configSpecs = require('./cfg-specs');
+
+function loadConfigFiles(configFilesBase, keysInOrder) {
+  var config = {};
+
+  var configFilePaths = keysInOrder.map(function(key) {
```

Comment:
Sorry, I meant that `fined` crawls the file system and only gives us the config file that matches.  Or are we using it differently?